lib/utils/js-parser.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
```

Comment:
FWIW, I would like for us to come up with a more specific mechanism for building up the parser within an Ember app.

Definitely doesn't change anything for now, but food for thought...

Reply:
Sure. What exactly were you thinking? Something that builds the parser dynamically? or just moving this sort of logic out to an addon?

Reply:
Yeah, something like that. TBH, I'm not sure, but it is _vaguely_ annoying to have to come up with this for every codemod separately.

Definitely something to think on...

Reply:
I'd be happy to spike something and ping you on it